README.md

```diff
@@ -0,0 +1,45 @@
+# Guide
+
+A website to teach Wagtail CMS to content editors, moderators and administrators.
+
+Wagtail guide brings you:
+- Tutorials, 
+- How-to, 
+- Reference 
+- Background information
+
+## Google Summer of Code
+
+This project is a part of Google Summer of Code 2022 under the organization [**Wagtail**](https://wagtail.org/).
+
+## Development
+
+### Frontend
+
+- Setup the appropriate version of node.    
+`nvm use`   
+The node version shoule be `v16.*`. You can check it by running 
+`node -v`   
+- Install all the dependencies     
+`yarn`  
+- Compile the frontend by running       
+`yarn start`    
+For production build run    
+`yarn start`    
+
+### Backend
+- Verify that your python version is `3.9.*` by running     
+`python -V` 
+- Setup virtual environment     
+    ``` bash
+    python -m venv env
+    source env/bin/activate
+    ```     
```

Comment:
I think the indent is not needed, is it?

Reply:
If you want to align the code block with the the other text, then yes it is needed. If I remove the indent, the code block will shift to left and will not align with bullets as they are indented.